### PR TITLE
Magento 2.4.0 Beta Release Note update 0603

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request (PR) updates the Magento 2.4.0 Beta release notes. This draft includes descriptions of fixed core issues and community and partner contribution tables. 

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-commerce.html
https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-open-source.html

whatsnew
Updates to the Magento 2.4.0 Beta Release Notes. 
https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-open-source.html
https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-commerce.html

### CODE REVIEW

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 1]
```diff
@@ -460,7 +509,23 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 <!--- MC-31481-->
 
-*  Magento now successfully imports customer data using the **Customer and Addresses (single file))** option when `cron` is enabled and the Customer Grid Indexer is set to **Update By Schedule**. After `cron` executes, the imported customer information is available in the Admin as expected. Previously, Magento imported the customer data, but did not update the customer grid with the newly imported customer records.
+*  Magento now successfully imports customer data using the **Customer and Addresses (single file)** option when `cron` is enabled and the Customer Grid Indexer is set to **Update By Schedule**. After `cron` executes, the imported customer information is available in the Admin as expected. Previously, Magento imported the customer data, but did not update the customer grid with the newly imported customer records.
+
+<!--- MC-31117-->
+
+*  You can now create a new customer from the storefront when date of birth is required. Previously, you could not create a new customer when this field was required, even when you entered valid DOB values. [GitHub-26700](https://github.com/magento/magento2/issues/26700)
```
Comment (keharper) [inline src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:516]:
```suggestion
*  You can now create a new customer from the storefront when the date of birth is required. Previously, you could not create a new customer when this field was required, even when you entered valid DOB values. [GitHub-26700](https://github.com/magento/magento2/issues/26700)
```

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 2]
```diff
@@ -648,6 +713,18 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 *  Magento now displays the **Credit memo** button after the partial refund of an order. Previously, Magento did not display this button after you created a partial refund, and you could not create a credit memo for the rest of the order.
 
+<!--- MC-31878-->
+
+*  Bulk order update through REST now updates order status as expected. Previously, Magento threw this error: `report.ERROR: Property "AdditionalInformation" does not have accessor method "setAdditionalInformation" in class "Magento\Sales\Api\Data\OrderPaymentInterface".`
```
Comment (keharper) [inline src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:718]:
```suggestion
*  Bulk order updates through REST now modify the order status as expected. Previously, Magento threw this error: `report.ERROR: Property "AdditionalInformation" does not have accessor method "setAdditionalInformation" in class "Magento\Sales\Api\Data\OrderPaymentInterface".`
```

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 3]
```diff
@@ -826,7 +921,7 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 <!--- MC--->
 
-*  The integration of third-party payment methods into the core Magento code has been deprecated. With this release, the integrations of the Authorize.Net, eWay, CyberSource, and Worldpay payment methods are deprecated. These core features are no longer supported and will be removed in the next minor release (2.4.0). Merchants should migrate to the official extensions that are available on the Magento Marketplace.
+*  The integration of third-party payment methods (Authorize.Net, eWay, CyberSource, and Worldpay) into the core Magento code has been deprecated. Merchants should migrate to the official extensions that are available on the Magento Marketplace.
```
Comment (keharper) [inline src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:924]: These have been removed now. Also Braintree

Authorize.Net and Braintree are in CE, so a modified version release note needs to be in the Open Source relnotes.

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 4]
```diff
@@ -968,6 +1093,8 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 ### Search
 
+*  Elasticsearch 7.6.x is now the default catalog search engine for Magento Commerce and Open Source. You cannot install or upgrade to Magento 2.4.0 without also installing Elasticsearch 7.6.x. Elasticsearch version 2.x code has been removed, and Elasticsearch versions 5.x and 6.x have been deprecated and are no longer supported. See [Elasticsearch]({{page.baseurl}}/install-gde/prereq/elasticsearch.html).
```
Comment (keharper) [inline src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:1096]:
```suggestion
*  Elasticsearch 7.6.x is now the default catalog search engine for Magento Commerce and Open Source. You cannot install or upgrade to Magento 2.4.0 without also installing Elasticsearch 7.6.x. Elasticsearch version 2.x code has been removed. Elasticsearch versions 5.x and 6.x have been deprecated and are no longer supported. See [Elasticsearch]({{page.baseurl}}/install-gde/prereq/elasticsearch.html).
```

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 5]
```diff
@@ -54,11 +54,11 @@ Starting with the release of Magento Commerce 2.3.2, Magento will assign and pub
...
 
-*  **Elasticsearch 7.x support**. The latest Elasticsearch 7.x version is now the default catalog search engine for Magento Commerce and Open Source. Elasticsearch version 2.x code has been removed, and Elasticsearch version 5.x has been deprecated. Elasticsearch v6.8 is still supported in Magento 2.4.0 (and 2.3.x). You cannot install or upgrade to Magento 2.4.0 without also installing Elasticsearch. See [Elasticsearch]({{page.baseurl}}/install-gde/prereq/elasticsearch.html).
+*  **Elasticsearch 7.6.x support**. Elasticsearch 7.6.x is now the default catalog search engine for Magento Commerce and Open Source. You cannot install or upgrade to Magento 2.4.0 without also installing Elasticsearch 7.6.x. Elasticsearch version 2.x code has been removed, and Elasticsearch versions 5.x and 6.x have been deprecated and are no longer supported. See [Elasticsearch]({{page.baseurl}}/install-gde/prereq/elasticsearch.html).
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:61]: changed.

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 6]
```diff
@@ -107,13 +107,15 @@ This release contains enhancements to core quality, which improve the quality of
...
+
 ### Adobe Stock Integration v2.0
 
 **Ability to license stock image previews from the Media Gallery**. Merchants can now find any Adobe Stock preview image in the Media Gallery, which reduces the number of steps required to license stock preview image.
 
-### Magento Media Gallery
+### New Media Gallery
 
-This replacement for the former Media Gallery offers a brand-new, searchable interface for Magento media assets. Administrators can now search, filter, and sort images up to 30x faster than they could in the earlier version of this feature. Merchants can use this tool to evaluate storefront image usage.
+This replacement for the former Media Gallery offers a new, searchable interface for Magento media assets. Administrators can now search, filter, and sort images up to 30x faster than they could in the earlier version of this feature. Merchants can use this tool to evaluate storefront image usage. Extension developers should be aware that extensions that were developed for the Media Gallery will not work as expected with the new Media Gallery.
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:118]: Kevin, not yet. Hopefully soon, after which I will link to the doc.

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 7]
```diff
@@ -54,11 +54,11 @@ Starting with the release of Magento Commerce 2.3.2, Magento will assign and pub
 
 The following platform upgrades help enhance website security and performance. Supported versions of PHP and PHPUnit, Elasticsearch, MySQL, and other dependencies are listed in [Magento 2.4 technology stack requirements]({{page.baseurl}}/install-gde/system-requirements-tech.html).
 
-*  **PHP 7.4 support introduced and PHP 7.1 and 7.2 deprecated**. Magento 2.4.0 introduces support for PHP 7.4. All testing of 2.4.0 has been done on PHP 7.4. You can still install Magento 2.4.x with PHP 7.3, but we recommend using PHP 7.4.
+*  **PHP 7.4 support introduced and PHP 7.1 and 7.2 deprecated**. Magento 2.4.0 introduces support for PHP 7.4. Magento 2.4.0 can be installed with Elasticsearch 7.6.x only.
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:57]: oops -- thanks. removed.

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md (hunk 1 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:516]: added "the"

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md (hunk 2 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:718]: changed

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md (hunk 3 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:924]: entered edit but kept all payment methods in OS notes -- PM says these are all open source.

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md (hunk 4 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:1096]: entered change.

File: src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md [hunk 8]
```diff
@@ -372,6 +511,10 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 *  Magento now displays all children of a selected parent category as expected. Previously, if you selected a parent category that is an anchor, but which did not have assigned products by itself, Magento did not display all nested products.
 
+<!--- MC-31171-->
+
+*  The Recently Viewed Products widget now works correctly when **Stores** > **Configuration** > **Catalog** > **Catalog** > **Recently Viewed/Compared** > **Synchronize widget products with backend storage** is set to **yes**.
```
Comment (keharper) [inline src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:516]:
```suggestion
*  The Recently Viewed Products widget now works correctly when **Stores** > **Configuration** > **Catalog** > **Catalog** > **Recently Viewed/Compared** > **Synchronize widget products with backend storage** is set to **Yes**.
```

Comment (jfrontain) [reply src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md:516]: changed

**Pull request merged**